runtime/service/npud/core/DBus.cc

```diff
@@ -86,5 +88,27 @@ gboolean DBus::on_handle_device_get_available_list(NpudCore *object,
   return TRUE;
 }
 
+gboolean DBus::on_handle_context_create(NpudCore *object, GDBusMethodInvocation *invocation,
+                                        gint arg_device_id, gint arg_priority)
+{
+  VERBOSE(DBus) << "on_handle_context_create with " << arg_device_id << ", " << arg_priority
+                << std::endl;
+  guint64 out_ctx;
+  int ret = -1;
+  // TODO Invoke Core function.
+  npud_core_complete_context_create(object, invocation, out_ctx, ret);
+  return TRUE;
```

Comment:
Don't we need to check return value (`ret`)?

Reply:
The `return TRUE` is from `gdbus` guideline. I wanted to find the document about this, but I couldn't. 
The real return value is `ret` of `npud_core_complete_context_create`.

Reply:
OK